### PR TITLE
Fix error when run azd init

### PR DESCRIPTION
**Pull request by rujche**
Follow-up of #22 . Fix error when run `azd init`.

> image

### CODE REVIEW

File: cli/azd/internal/auth_type.go [hunk 1]
```diff
@@ -6,9 +6,23 @@ type AuthType string
...
+
+func GetAuthTypeDescription(authType AuthType) string {
+	switch authType {
+	case AuthTypeUnspecified:
+		return "Unspecified"
+	case AuthTypePassword:
+		return "Username and password"
+	case AuthTypeConnectionString:
+		return "Connection string"
+	case AuthTypeManagedIdentity:
```
Comment (saragluna) [inline cli/azd/internal/auth_type.go:24]: If we use umi here, maybe a const for umi, a const for smi?

File: cli/azd/internal/repository/app_init.go [hunk 2]
```diff
@@ -560,54 +560,81 @@ func (i *Initializer) prjConfigFromDetect(
...
-
-			for {
-				dbName, err := promptDbName(i.console, ctx, database)
-				if err != nil {
-					return config, err
+				resourceConfig = project.ResourceConfig{
+					Type: project.ResourceTypeDbCosmos,
+					Name: databaseName,
+					Props: project.CosmosDBProps{
+						DatabaseName: databaseName,
```
Comment (saragluna) [inline cli/azd/internal/repository/app_init.go:605]: Is this database name the name of the database server, or a database?

File: cli/azd/internal/auth_type.go (hunk 1 above)
Comment (rujche) [reply cli/azd/internal/auth_type.go]: 1. Rename to umi.
2. Not ass smi in current PR, we can add it when smi is used.

File: cli/azd/internal/repository/app_init.go (hunk 2 above)
Comment (rujche) [reply cli/azd/internal/repository/app_init.go:605]: It's name of database. Now we do not have the information of database server name. The server name is '${abbrs.dBforMySQLServers}${resourceToken}'.  How about using const string for each database? For example: for cosmos db, use "cosmosdb", for MySql, use "mysql"

Comment (rujche) [reply cli/azd/internal/repository/app_init.go:605]: Updated.

**Review by saragluna — APPROVED** [review]

File: cli/azd/internal/repository/app_init.go (hunk 2 above)
Comment (saragluna) [reply cli/azd/internal/repository/app_init.go:605]: No need to update this, let's keep the random name now for some services have constraints on global unique service name. I was just curious about where the database name is used for.

**Pull request merged**